src/dbnode/storage/shard.go

```diff
@@ -2999,14 +3009,18 @@ func (s *dbShard) logFlushResult(r dbShardFlushResult) {
 	)
 }
 
-func (s *dbShard) finishWriting(startTime time.Time, nextVersion int) error {
+func (s *dbShard) finishWriting(
+	startTime time.Time,
+	nextVersion int,
+	setInitialFileOpStatus bool,
```

Comment:
nit: this is technically marking the warm flush state as success, should we change the var name to something like `markWarmFlushStateSuccess` (I copied the name of the fn that does that)?

Reply:
Renamed as suggested.